mirobo/airpurifier.py

```diff
@@ -0,0 +1,63 @@
+from .device import Device
+from .containers import AirPurifierStatus
+
+
+class AirPurifier(Device):
+    """Main class representing the air purifier."""
+
+    def status(self):
+        """Retrieve properties."""
+
+        # A few more properties:
+        properties = ['power', 'aqi', 'humidity', 'temp_dec',
+                      'mode', 'led', 'led_b', 'buzzer', 'child_lock',
+                      'limit_hum', 'trans_level', 'bright',
+                      'favorite_level', 'filter1_life', 'act_det',
+                      'f1_hour_used', 'use_time', 'motor1_speed']
+
+        values = self.send(
+            "get_prop",
+            properties
+        )
```

Comment:
To add, do you happen to have an example also for this case? It would be nice to have example answers for all calls supported by devices.